.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  linux:
+    name: go-linux
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: "1.22"
```

Comment:
Should we test against multiple versions of Go? CRDB is only currently on 1.21.

Here's an example of that with https://github.com/cockroachdb/apd/blob/master/.github/workflows/go.yml#L15. It also shows how to test on multiple CPU architectures, for later.

Reply:
Done. I added testing for go1.20 through go1.22.